### ISSUE TITLE
Lost / missing alt text when you send toot

### ISSUE THREAD

**Issue by seachdamh** [NicolasConstant/sengi#430]
Bug: When write toot with attached image and alt text hitting enter in alt text area send toot with image but without alt text. 

Reproduction: 

1. Add image
2. Add alt text (description) for image.
3. Hit enter while in alt text edit box sending toot
Result: toot will be send withot alt text/descrioption

It would be better to fix it so sending toot with "enter" is
a) imposible - but it is not good from point of view usability and user user experience
b) send toot, image with alt text (as it shuld work from user point of view)
